From patchwork Mon Jan  4 00:27:48 2021
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Rosen Penev <rosenp@gmail.com>
X-Patchwork-Id: 1421968
Return-Path: 
 <openwrt-devel-bounces+incoming=patchwork.ozlabs.org@lists.openwrt.org>
X-Original-To: incoming@patchwork.ozlabs.org
Delivered-To: patchwork-incoming@bilbo.ozlabs.org
Authentication-Results: ozlabs.org;
 spf=pass (sender SPF authorized) smtp.mailfrom=lists.openwrt.org
 (client-ip=2001:8b0:10b:1231::1; helo=merlin.infradead.org;
 envelope-from=openwrt-devel-bounces+incoming=patchwork.ozlabs.org@lists.openwrt.org;
 receiver=<UNKNOWN>)
Authentication-Results: ozlabs.org;
 dmarc=fail (p=none dis=none) header.from=gmail.com
Authentication-Results: ozlabs.org;
	dkim=pass (2048-bit key;
 secure) header.d=lists.infradead.org header.i=@lists.infradead.org
 header.a=rsa-sha256 header.s=merlin.20170209 header.b=oI28Y4rz;
	dkim=fail reason="signature verification failed" (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.a=rsa-sha256
 header.s=20161025 header.b=olQb0VsB;
	dkim-atps=neutral
Received: from merlin.infradead.org (merlin.infradead.org
 [IPv6:2001:8b0:10b:1231::1])
	(using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
	 key-exchange X25519 server-signature RSA-PSS (4096 bits) server-digest
 SHA256)
	(No client certificate requested)
	by ozlabs.org (Postfix) with ESMTPS id 4D8Ghq1Yvxz9s24
	for <incoming@patchwork.ozlabs.org>; Mon,  4 Jan 2021 11:29:13 +1100 (AEDT)
DKIM-Signature: v=1; a=rsa-sha256; q=dns/txt; c=relaxed/relaxed;
	d=lists.infradead.org; s=merlin.20170209; h=Sender:Content-Transfer-Encoding:
	Content-Type:List-Subscribe:List-Help:List-Post:List-Archive:List-Unsubscribe
	:List-Id:MIME-Version:Message-Id:Date:Subject:To:From:Reply-To:Cc:Content-ID:
	Content-Description:Resent-Date:Resent-From:Resent-Sender:Resent-To:Resent-Cc
	:Resent-Message-ID:In-Reply-To:References:List-Owner;
	bh=1BS+j5cOIbjpKKLlqnyUj5zivavw6/LBAt8JvOmJ0PY=; b=oI28Y4rzlwLMWZh0WW+g1fOkbF
	L0gy7Ee0F26rOWrCv9/8eKa9cflFM3GyGA+pIQVssA9vI9JDOD9N06nyo3r1e6RDzHAS3rPntQ40h
	TTozBD0hkJ7UTwwXk4eeBqA695UiMcDy+4HlrC2eIq03SZJP6+TRlxdWw/s2iJUgLuKqvr0HqkiCt
	tlN26PRNYIqe3IraQc+mKbppVSCF9MFYRMF+4rq4YRAu33QQ5UxCU7VmhA95C786jdck7nsZ457lO
	pAAvIML4VTJcwO6BfYz9TZMYqFpSKZeB6d1BE3M8QpLRQegKKg8NnimzMX4y86tW00UKr/9kXPKBC
	K15gJLoA==;
Received: from localhost ([::1] helo=merlin.infradead.org)
	by merlin.infradead.org with esmtp (Exim 4.92.3 #3 (Red Hat Linux))
	id 1kwDj3-0007Aj-Bo; Mon, 04 Jan 2021 00:27:57 +0000
Received: from mail-pg1-x529.google.com ([2607:f8b0:4864:20::529])
 by merlin.infradead.org with esmtps (Exim 4.92.3 #3 (Red Hat Linux))
 id 1kwDiy-00078t-DG
 for openwrt-devel@lists.openwrt.org; Mon, 04 Jan 2021 00:27:53 +0000
Received: by mail-pg1-x529.google.com with SMTP id v19so17663093pgj.12
 for <openwrt-devel@lists.openwrt.org>; Sun, 03 Jan 2021 16:27:51 -0800 (PST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20161025;
 h=from:to:subject:date:message-id:mime-version
 :content-transfer-encoding;
 bh=DMpIamJdLnyBQbfzRmSLfzt8u/dkV7ptNxH24M6blXk=;
 b=olQb0VsBOYIxRRiEenyVGk0rCtavmjW/2IQBHApMUK5qgGN/pOSv6PvKOL1M5NG8j/
 2p6iv+dAYsXK5T2fQb4jzWwMjdH+LelYzNJmnEm0ppYQa8HVJ0kazPdTNLjtciD//4x+
 /xm4+YdXj/F16cvEomJ2AncHHVbLBoSqKIqnVE6xg0fTz42rMbmoTG7iX/cBXUic/fbM
 rdH7ksOJIxWmQgI6amd17fnSFrR7AaAPANKUdSqu7RWxDO+6QherfZLpmhCsALFbhGUC
 SOV/ax9KwxUqzqq2ibLm+TDhyUlQ3foWVVpTWe8/+StMvBlNCPGQ7Yu7M4LfHB0S2Uli
 yOLw==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
 d=1e100.net; s=20161025;
 h=x-gm-message-state:from:to:subject:date:message-id:mime-version
 :content-transfer-encoding;
 bh=DMpIamJdLnyBQbfzRmSLfzt8u/dkV7ptNxH24M6blXk=;
 b=AbANqtDo3qWndeUTijUQPmZxa/hu14e/OB/q+sxm2vreOU+6SEFfPQC8VlYaVX675l
 wEEGxMnmzBXSZIYg+piG2NAYb9qwU0iCAVC266xyAfFpUTTveT/rsV5QXoURZ5nNCM4G
 eEAzbsKM/ztv8SIrfGvW2MgLyNrtlQYUr12ze+43k33PEuBRxqCSTqN1LUQtRbmP4kK0
 ru85MA0Q+kQ3zZoMStJyzSq5GRImxjK5WgItrKomqKVT8pOHLzMphd442R5FSmuFewMI
 a0segM3mlVBXIxfEvZKvduvrSXi614v/CbarMtaafwzo0kBtwadyWkln/TXnkkP3kqQC
 ySnA==
X-Gm-Message-State: AOAM530eCHOprHZ6flApAWICjOosmOd7iMkYGPJ9miJq+eh3GmVM693j
 A2g1m786yd3/IB3NI5syGH8E+4acProIww==
X-Google-Smtp-Source: 
 ABdhPJw2UpS13E4yV/QSGt/hVt6CvvFOpJX+woPL/sTml5A3sJjxK85Gkmz2ZvKopTmJOCDW5q2wMg==
X-Received: by 2002:aa7:9afb:0:b029:19e:45d8:b102 with SMTP id
 y27-20020aa79afb0000b029019e45d8b102mr63511278pfp.48.1609720070129;
 Sun, 03 Jan 2021 16:27:50 -0800 (PST)
Received: from mangix-trapnet.lan ([2001:470:1f05:79e::a89])
 by smtp.gmail.com with ESMTPSA id nm6sm18802171pjb.25.2021.01.03.16.27.49
 for <openwrt-devel@lists.openwrt.org>
 (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
 Sun, 03 Jan 2021 16:27:49 -0800 (PST)
From: Rosen Penev <rosenp@gmail.com>
To: openwrt-devel@lists.openwrt.org
Subject: [PATCH] gmp: update to 6.2.1
Date: Sun,  3 Jan 2021 16:27:48 -0800
Message-Id: <20210104002748.762138-1-rosenp@gmail.com>
X-Mailer: git-send-email 2.29.2
MIME-Version: 1.0
X-CRM114-Version: 20100106-BlameMichelson ( TRE 0.8.0 (BSD) ) MR-646709E3 
X-CRM114-CacheID: sfid-20210103_192752_594060_DD721D5D 
X-CRM114-Status: UNSURE (   9.02  )
X-CRM114-Notice: Please train this message.
X-Spam-Score: -0.2 (/)
X-Spam-Report: SpamAssassin version 3.4.4 on merlin.infradead.org summary:
 Content analysis details:   (-0.2 points)
 pts rule name              description
 ---- ----------------------
 --------------------------------------------------
 -0.0 RCVD_IN_DNSWL_NONE     RBL: Sender listed at https://www.dnswl.org/,
 no trust [2607:f8b0:4864:20:0:0:0:529 listed in]
 [list.dnswl.org]
 0.0 FREEMAIL_FROM          Sender email is commonly abused enduser mail
 provider [rosenp[at]gmail.com]
 -0.0 SPF_PASS               SPF: sender matches SPF record
 0.0 SPF_HELO_NONE          SPF: HELO does not publish an SPF Record
 0.1 DKIM_SIGNED            Message has a DKIM or DK signature,
 not necessarily
 valid
 -0.1 DKIM_VALID Message has at least one valid DKIM or DK signature
 -0.1 DKIM_VALID_AU          Message has a valid DKIM or DK signature from
 author's domain
 -0.1 DKIM_VALID_EF          Message has a valid DKIM or DK signature from
 envelope-from domain
X-BeenThere: openwrt-devel@lists.openwrt.org
X-Mailman-Version: 2.1.29
Precedence: list
List-Id: OpenWrt Development List <openwrt-devel.lists.openwrt.org>
List-Unsubscribe: <https://lists.openwrt.org/mailman/options/openwrt-devel>,
 <mailto:openwrt-devel-request@lists.openwrt.org?subject=unsubscribe>
List-Archive: <http://lists.openwrt.org/pipermail/openwrt-devel/>
List-Post: <mailto:openwrt-devel@lists.openwrt.org>
List-Help: <mailto:openwrt-devel-request@lists.openwrt.org?subject=help>
List-Subscribe: <https://lists.openwrt.org/mailman/listinfo/openwrt-devel>,
 <mailto:openwrt-devel-request@lists.openwrt.org?subject=subscribe>
Sender: "openwrt-devel" <openwrt-devel-bounces@lists.openwrt.org>
Errors-To: 
 openwrt-devel-bounces+incoming=patchwork.ozlabs.org@lists.openwrt.org

Fix license information.

Signed-off-by: Rosen Penev <rosenp@gmail.com>
---
 package/libs/gmp/Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/package/libs/gmp/Makefile b/package/libs/gmp/Makefile
index 4ea69019f6..eb7d808139 100644
--- a/package/libs/gmp/Makefile
+++ b/package/libs/gmp/Makefile
@@ -8,17 +8,17 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=gmp
-PKG_VERSION:=6.1.2
-PKG_RELEASE:=2
+PKG_VERSION:=6.2.1
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)$(PKG_REVISION).tar.xz
 PKG_SOURCE_URL:=@GNU/gmp/
-PKG_HASH:=87b565e89a9a684fe4ebeeddb8399dce2599f9c9049854ca8c0dfbdea0e21912
+PKG_HASH:=fd4829912cddd12f84181c3451cc752be224643e87fac497b69edddadc49b4f2
 
 PKG_BUILD_PARALLEL:=1
 PKG_INSTALL:=1
 PKG_FIXUP:=autoreconf
-PKG_LICENSE:=GPL-2.0+
+PKG_LICENSE:=GPL-2.0-or-later
 
 PKG_USE_MIPS16:=0
 
